src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/MethodCodeNode.cs

```diff
@@ -29,7 +29,7 @@ public MethodCodeNode(MethodDesc method)
             Debug.Assert(method.GetCanonMethodTarget(CanonicalFormKind.Specific) == method);
             _method = method;
         }
-
+        
```

Comment:
Nit: unnecessary white space getting checked in. There's no reason to touch this file now.